horton-michael/movie-night-planner
# 66 pass in title from tmdb data to the ombd api in order to fetch all the more detailed data

## Purpose
Fixes #[issue]

Feeds data from the first API request from TMDB to the second API request OMDB. Accounts for the new data structure when genre is added, may need to update with new changes. adds call to function inside of first API call function. Console logs results. Still only searches once based on last movie in the array (still needs fixing)

also deletes pesky additional script file


## Changes

Please provide an overview of code changes, especially key files and areas of significant change.

code changes are on script.js  significant change to the getOmdbData() function, one change to tmdb function to call the next function with the data


## Steps to Reproduce or Test

when you click a genre in the pull down menu, both functions should run  and console log the up-to-data
